database/utils/random.js

```diff
@@ -0,0 +1,14 @@
+
+/**
+ * 
+ * @param {Number} min Lower bound, inclusive
+ * @param {Number} max Upper bound, inclusive
```

Comment:
max is exclusive according to the comment below?

Reply:
Ahh yea, I forgot to delete that. Thanks.